src/plugins/data/server/search/saved_objects/search_session.ts

```diff
@@ -70,6 +58,9 @@ export const searchSessionSavedObjectType: SavedObjectsType = {
       version: {
         type: 'keyword',
       },
+      isCanceled: {
```

Comment:
Do we need `isCanceled`? I don't think "cancel" is an action you can take from the management menu, and cancelling from the pop-up deletes the session.

Reply:
I explained my thinking in the description: 

> I had to introduce isCanceled flag to remove status in the saved object because it was used for Status.Canceled; we have this feature implemented on the server, but we don't use it on UI: we always delete the session saved object. 
I also didn't want to get rid of the feature completely because I think we actually should stop deleting the sessions in some cases and instead "soft delete" them with isCanceled flag. 

So I decided to keep this feature server-side even though for now it is not used from the UI. 
But I think longer-term we should actually start using it and this implementation is very low-maintance. 
WDYT?

Reply:
Are you saying we'd do this when a user deletes from the management screen?

Reply:
I think we should cancel the session when user stops the running session while it is running inside the app and we should give option to cancel a running session on mgmt screen. 
Then delete should be a separate option on mgmt screen, but we should never delete without explicit action from user